packages/checkbox/test/checkbox.test.ts

```diff
@@ -238,6 +238,13 @@ describe('Checkbox', () => {
         expect(el.checked).to.be.true;
     });
 
+    it('has name attribute', () => {
+        let el = testFixture.querySelector('#checkbox0') as Checkbox;
+
+        el = testFixture.querySelector('#checkbox1') as Checkbox;
+        expect(el.hasAttribute('name'));
+    });
```

Comment:
Could you also include a test for the expected value of `name` if `name = undefined`?

Reply:
Also, what is the value of `el.name` here?

Reply:
Also, what is the value of `el.name` here?

Reply:
the value here is undefined because  it's not yet initialised

Reply:
Added 2 extra tests here for the same